lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
   return (
     <div
-      className={''}
+      className={ props.tab === props.selectedTab ?'tab active-tab' : 'tab'}
```

Comment:
a really nice looking conditional rendering based on whether something is true. Good job. You will continually reuse this pattern so mastering this early is always a good thing